test/loading.spec.js

```diff
@@ -291,4 +291,23 @@ describe('Loading Monkey', function () {
       spans.length.should.equal(4);
     });
   });
+
+  it('should generate letters for names with spaces (slow)', function () {
+    var monkey = new Monkey($('<div />').attr('data-key', 'lmn-book'), {
+      letters: true,
+      book: {
+        name: 'Lee T',
+        gender: 'girl',
+        locale: 'en-GB'
+      }
+    });
+
+    return monkey.promise
+    .then(Monkey.letters._generateHtml(true, options.lang, true))
```

Comment:
i just refactored this, would recommend rebasing onto master before you do anything else